lib/mayaHydra/mayaPlugin/renderOverride.cpp

```diff
@@ -941,37 +941,38 @@ void MtohRenderOverride::SelectionChanged(
     //    reading the Maya selection must be done from the Maya selection
     //    changed callback, not the UFE selection changed callback.
     using SnOp = Ufe::SelectionCompositeNotification::Op;
-    static auto appendSn = [this](const SnOp& op) {
-        _selectionSceneIndex->AddSelection(op.item->path());
```

Comment:
Capturing the selection scene index once, on creation of the static functor, is incorrect, as the selection scene index is destroyed and re-created.  We were therefore calling member functions on a destroyed object, causing crashes.